sys/src/main/java/com/alibaba/nacos/sys/utils/InetUtils.java

```diff
@@ -219,35 +221,46 @@ public static boolean isIP(String str) {
         return matcher.matches();
     }
     
+    /**
+     * juege str is right domain.
+     *
+     * @param str nacosIP
+     * @return nacosIP is domain
+     */
+    public static boolean isDomain(String str) {
+        InetSocketAddress address = new InetSocketAddress(str, 0);
+        return !address.isUnresolved();
```

Comment:
A Warn log can be output here to indicate that the domain name is not reachable

Reply:
solve it